Engine/Module/ModuleRender.h

```diff
@@ -63,6 +72,7 @@ class ModuleRender : public Module
 	void* context = nullptr;
 
 	OLQuadTree ol_quadtree;
+	OLAABBTree* ol_abbtree = new OLAABBTree(INITIAL_SIZE_AABBTREE);
```

Comment:
Why a pointer? Also, as default initialize pointers with `nullptr`

Reply:
It was a pointer for simplicity because is easier to just call the destructor by deleting the pointer and then creating a new one. Maybe i could call the destructor explicitly but we can talk about it.